### PR TITLE
feat: 🎸 create QR code

### PR DESCRIPTION
**Pull request by zim314**

### CODE REVIEW

**Comment by blp100** [comment]
Screenshot 2024-07-03 at 11 58 05 PM

File: src/app/member-center/ticket-inquiry/api/qrcode/[ticketID]/route.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { fetchAPI } from '@action/utils';
+
+export async function PUT(ticketID: string) {
```
Comment (blp100) [inline src/app/member-center/ticket-inquiry/api/qrcode/[ticketID]/route.ts:3]: request, 也必須是Request

ref: https://nextjs.org/docs/app/building-your-application/routing/route-handlers#request-body

Comment (zim314) [reply src/app/member-center/ticket-inquiry/api/qrcode/[ticketID]/route.ts]: 終於看懂了！

**Review by blp100 — APPROVED** [review]
應該沒問題了，手機掃描有拿到qrcode

**Pull request merged**